tensorflow/python/ops/nn.py

```diff
@@ -88,6 +88,7 @@
 @@ctc_beam_search_decoder
 @@top_k
 @@in_top_k
+@@nth_element
```

Comment:
New symbols should be introduced in contrib first, and only moved to core tensorflow after their API has stabilized a little.

Removing this line will make the API test failure go away.

Reply:
Thank you very much. I have removed this line.

Reply:
Right, so let's add a similar line to tf.contrib.nn (and import this symbol from there) so users can use the nth_element op.

Reply:
Sure, so should I delete the file of `tensorflow/python/ops/nth_element_op.py` and other relative modifies, build a new directory of `contrib/nth_element/` and move those modifies to `contrib/nth_element/`??